packages/curve-ui-kit/src/themes/typography/mui-typography.ts

```diff
@@ -1,22 +1,45 @@
 import type { Components } from '@mui/material/styles'
+import { TypographyVariantKey } from './create-typography'
+
+const variantMapping = {
+  headingXxl: 'h1',
+  headingMBold: 'h2',
+  headingMLight: 'h3',
+  headingSBold: 'h4',
+  headingXsBold: 'h5',
+  headingXsMedium: 'h6',
+
+  bodyMRegular: 'p',
+  bodyMBold: 'p',
+  bodySRegular: 'p',
+  bodySBold: 'p',
+  bodyXsRegular: 'p',
+  bodyXsBold: 'p',
+
+  buttonXs: 'span',
+  buttonS: 'span',
+  buttonM: 'span',
+
+  tableHeaderM: 'dfn',
+  tableHeaderS: 'dfn',
+  tableCellL: 'span',
+  tableCellMRegular: 'span',
+  tableCellMBold: 'span',
+  tableCellSRegular: 'span',
+  tableCellSBold: 'span',
+
+  highlightXsNotional: 'em',
+  highlightXs: 'em',
+  highlightS: 'em',
+  highlightM: 'strong',
```

Comment:
Are you sure about using `em` and `strong` here? There's two issues:

1. `xs` and `xsnotional` are now also italic, because by default browsers apply italic styling to `<em>` tags. 
2. If you *do* decide to keep `strong`, you no longer need `fontWeight: 'Bold'` in `TYPOGRAPHY_VARIANTS` as browser apply a `font-weight: bold` by default to `<strong>` elements.

![image](https://github.com/user-attachments/assets/b390658d-1fe9-4134-ad2f-67232c425044)

Reply:
yeah I was thinking about semantic HTML but this is a PITA. I'll use `<p>`